tensorflow/contrib/rnn/python/kernel_tests/rnn_test.py

```diff
@@ -0,0 +1,215 @@
+"""Tests for rnn module."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import time
+import timeit
+
+import numpy as np
+from six.moves import xrange  # pylint: disable=redefined-builtin
+
+import tensorflow as tf
+from tensorflow.python.kernel_tests import rnn_test
+
+tf.contrib.rnn.Load()
```

Comment:
Not sure if we discussed this... why can't we just Load() inside the __init__ for tf.contrib.rnn?